pandas/tests/dtypes/test_generic.py

```diff
@@ -1,3 +1,4 @@
+import random
```

Comment:
is the import and setting the seed below required?

Reply:
will check and remove if not required